src/config/configJsonSchema.js

```diff
@@ -141,8 +257,19 @@ module.exports = {
       format: 'ipv4',
     },
     network: {
-      type: 'string',
-      enum: Object.values(NETWORKS),
+      type: 'object',
+      properties: {
+        name: {
+          type: 'string',
+          enum: Object.values(NETWORKS),
+        },
+        version: {
+          type: 'integer',
```

Comment:
It's optional, so let's make it `type: ['string', 'null'],` as other optional params